client/stylesheets/_toCat.scss

```diff
@@ -2875,8 +2879,91 @@ i:hover {
 
 .managerOfferButton ul li a:hover {
     background-color: #f73c37;
+} */
+
+.managerOfferButton {
+    margin-top: 20px !important;
+}
+
+.optionsCardItems li a:hover {
+	background-color: #d3d3d3 !important;
```

Comment:
Color literals like `#d3d3d3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs